packages/mobile/src/components/countdown/countdown.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import { View } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { useCountdownInterval, useTranslation } from '~/hooks/hooks';
+import { CountdownSegment } from './components/components';
+import { SegmentPosition } from './common/enums/enums';
+import { styles } from './styles';
+
+type Props = {
+  endDate: Date;
+};
```

Comment:
```suggestion
type CountdownProps = Pick<ProductDto, 'endDate'>;
```
And more likely endDate will be a string (I assume backend guys have typed it wrong)